lib/unimatrix/authorization/filters/requires_policies.rb

```diff
@@ -80,7 +80,8 @@ def policies
     end
 
     def retrieve_policies( resource_name, access_token, realm )
-      if resource_name && access_token && realm
+      if resource_name && access_token
+        realm    = realm || '*'
```

Comment:
Because the /realms endpoint returns all policies/realms an user has access took. There is no realm uuid to check against.
This is the search query `'realm/*::regent::realms/*'`. The above is so that `requires_policies :realms` doesn't throw error.